TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md

```diff
@@ -0,0 +1,122 @@
+
+Fritz Alder (FA) asked for clarification on the primary use case the project intends to address. Mingshen Sun (MS) reviewed the primary use case and provided additional technical details. Discussion ensued
+
+DM asked for clarification on which portion of the “Two-Stage” architecture was included in the project. MS stated that both the programming and execution stages were included. DM asked what the differences were in functionality between this and Google Confidential Space. MS provided technical details on the differences. DM lastly asked how attestation is or is not used in the project. MS stated that it is and provided context on the scenarios where it is used.
+
+Alex Fernandez (AF) asked if the token met the CCC definition of "hardware-based, attested Trusted Execution Environment"? AF asked DM if we should ask the folks at Google if this meets the definition.
+
+Discussion ensued.
+
+DM asked if someone from the TAC wanted to make a motion to vote on the project or if folks wanted more time to consider the project. AF stated that he would prefer more time and to discuss privately with TAC members before voting. DM stated that it would be kept on the agenda and encouraged TAC members to use existing channels to discuss. Fritz Alder and Manu Fontaine agreed that they would like more time for discussion. Vini Jaisal (VJ) offered to reach out to any members that require any further clarification or information.
+
+## TAC Objectives / TAC Goals
+
+DM introduced Henry Wang (HW) and Kenvin Hui (KH) to discuss TAC Goals
+HW announced that they have an internal Attestation encrypted channel establishment library project base on C++ helper functions that they would be interested in open sourcing. They are still working on removing internal dependencies before it is ready. Aiming to open source it before the end of 2024.
+DM welcomed the proposal and said the submission is welcome and does not have to be a finished product, leaving room for contributions.
+
+KH discussed starting a working group around open source code transparency for data center owner companies. They have already discussed this with the folks from Project Oak. AF asked if they have looked at the documentation around Asure code transparency service and how this would be different from this proposal.KH he briefly looked at it. The aim of the group would be to bring everyone’s code transparency work together. Started the working group and would aim to bring more roleplayers together.
+AF also indicated he would be interested in this work.
+
+DM pointed out that this idea is helpful, but it produces more goals to pursue rather than addressing goals that were already set by the TAC of 2024. He further proposed that KH can start with a blog post figuring out what is already available in the field. That could also be presented as a TAC Tech talk to get more community input about other areas where this is valid.
```

Comment:
by the TAC for 2024.

That could also be presented as a TAC Tech talk to get more community input about other areas where this is valid.

suggested rewording of "That could also be presented as a TAC Tech talk to get more community input about other areas where this is valid."

in the field suggesting a TAC tech talk to solicit community feedback.

Reply:
agree